src/plugins/in-app-menu/renderer/TitleBar.tsx

```diff
@@ -270,8 +270,21 @@ export const TitleBar = (props: TitleBarProps) => {
 
     // tracking mouse position
     window.addEventListener('mousemove', listener);
+    const ytmusicAppLayout = document.querySelector<HTMLElement>('#layout');
+    ytmusicAppLayout?.addEventListener("scroll",()=>{
+    const scrollValue = ytmusicAppLayout.scrollTop;
+    if (scrollValue > 20){
+      ytmusicAppLayout.classList.add("content-scrolled");
+    }
+    else{
+      ytmusicAppLayout.classList.remove("content-scrolled");
+    }
+    })
+
```

Comment:
what is the function of `.content-scrolled`? I can't find `content-scrolled` class in `titlebar.css` file

Reply:
This class is added/removed when you scroll the app. What I did is I made only the app layout scrollable and the whole app (only when the inappmenu is active) and this class serves to change the nav bars(top and side) background color. The class is already built by YouTube it's not new, that's why it's not present in the code.
Try the code and activate in app menu, scroll and watch the nav bars background color you'll get it.